torchinductor/utils.py

```diff
@@ -165,3 +165,39 @@ def wrapper(self):
         return getattr(self, key)
 
     return wrapper
+
+
+def has_incompatible_cudagraph_ops(gm):
+    forbidden_list = set(
+        [
+            "fbgemm.dense_to_jagged.default",
+            "fbgemm.jagged_to_padded_dense.default",
+        ]
+    )
+    for node in gm.graph.nodes:
+        if str(node.target) in forbidden_list:
+            return True
+    return False
+
+
+def assert_size_match(item, size):
+    assert isinstance(size, tuple)
+    if size == ():
```

Comment:
this seems more relaxed than previous version, is this intended?

Reply:
I was not sure what it means to check if size is () ... let me try to change this back and see what I hit.

Reply:
@jansel when do we want to do a size == () check?

Reply:
What is item in this case?

Reply:
A tensor

Reply:
Is the tensor 0D?  If so won't the normal assert work?

Reply:
Previously when size was `()` and tensor was not 0d, we would error out on assert. Here it looks like we just pass it.